app/src/main/java/org/simple/clinic/activity/TheActivity.kt

```diff
@@ -64,19 +67,12 @@ class TheActivity : AppCompatActivity() {
   }
 
   override fun attachBaseContext(baseContext: Context) {
+    setupDiGraph()
```

Comment:
This was needed to inject the locale before the `ScreenRouter` gets setup so that the context could be overriden with the selected `Locale`.